test/pensions/config/childrenInformation.unit.spec.jsx

```diff
@@ -50,7 +53,8 @@ describe('Child information page', () => {
     );
     const formDOM = getFormDOM(form);
     formDOM.submitForm(form);
-    expect(formDOM.querySelectorAll('.usa-input-error').length).to.equal(4);
+
+    expect(formDOM.querySelectorAll('.usa-input-error').length).to.equal(3);
```

Comment:
Previously, the date passed in (`undefined`) was causing `isEligibleForDisabilitySupport()` to return true, so the `disabled` field was showing up when it shouldn't have. That's why we went from 4 to 3 required fields.